redfish-core/src/redfish.cpp

```diff
@@ -253,6 +253,10 @@ RedfishService::RedfishService(App& app)
 
     // Note, this must be the last route registered
     requestRoutesRedfish(app);
+
+#ifdef BMCWEB_ENABLE_HW_ISOLATION
+    requestRoutesSystemHardwareIsolationLogService(app);
```

Comment:
I think you have to move this up or you get problems.. See the comment on line 254

Reply:
done